docs/README.md

```diff
@@ -4,6 +4,7 @@ Interacting with the Substra platform:
 
 - [Command line interface](../references/cli.md)
 - [SDK](../references/sdk.md)
+- [Ingest data samples](./ingest_data.md)
```

Comment:
I feel this link should be under "Learning about the substra platform".

It should also be included in the repo's main README.

Reply:
Yes, thanks.